### PR TITLE
Update fileviews.md

### PR DESCRIPTION
**Pull request by ychae**
added a section on embedding file views to wikis

### CODE REVIEW

**Comment by sgosline** [comment]
Are there plans to update this to apply to cross-project views?

**Comment by ychae** [comment]
@sgosline Do you mean adding in examples for embedding cross-project views into wikis?

**Comment by sgosline** [comment]
Someone pointed me to this PR as documentation for file views, but now I see that it's just about the wiki embedding.  I'd like to see the document review to multiple projects, will create an issue.  Carry on with merging this :-D

File: articles/fileviews.md [hunk 1]
```diff
@@ -64,6 +64,20 @@ In advanced search, you can use a SQL-like query to search for files in that vie
 
 <img id="image" src="/assets/images/fileViewAdvancedSearch.png">
 
+## Insert a View into a Wiki
+File views an also be placed inside a `Wiki` once they've been created. You can embed the entire file view or a subset of a query on it.
```
Comment (kdaily) [inline articles/fileviews.md]: link to section on Wiki widgets

**Pull request merged**